cmd/templ/generatecmd/eventhandler.go

```diff
@@ -80,71 +79,73 @@ type FSEventHandler struct {
 	fileNameToLastModTimeMutex *sync.Mutex
 	fileNameToError            map[string]struct{}
 	fileNameToErrorMutex       *sync.Mutex
+	fileNameToOutput           map[string]generator.GeneratorOutput
+	fileNameToOutputMutex      *sync.Mutex
+	devMode                    bool
 	hashes                     map[string][sha256.Size]byte
 	hashesMutex                *sync.Mutex
 	genOpts                    []generator.GenerateOpt
 	genSourceMapVis            bool
-	DevMode                    bool
 	Errors                     []error
 	keepOrphanedFiles          bool
 	writer                     func(string, []byte) error
 	lazy                       bool
 }
 
-func (h *FSEventHandler) HandleEvent(ctx context.Context, event fsnotify.Event) (goUpdated, textUpdated bool, err error) {
+func (h *FSEventHandler) HandleEvent(ctx context.Context, event fsnotify.Event) (updated, goUpdated, textUpdated bool, err error) {
```

Comment:
Is it worth this returning a struct now that there is 3 non-error reture values?

Reply:
This should make the returns easier to read:

```go
if h.devMode {
	// Don't delete the file if we're in dev mode, but mark that text was updated.
	result.TextUpdated = true
	return result, nil
}
```
or
```go
if h.devMode {
	// Don't delete the file if we're in dev mode, but mark that text was updated.
	result := EventResult{ TextUpdated: true }
	return result, nil
}
```